src/components/DesignTheme/View/PersonaView.jsx

```diff
@@ -410,6 +411,23 @@ const PersonaView = ({ content }) => {
                 </div>
               </article>
             ) : null}
+            {content.relatedItems?.length > 0 ? (
+              <article
+                id="related-items"
+                className="it-page-section anchor-offset mt-5"
+              >
+                <h4>{intl.formatMessage(messages.related_items)}</h4>
+                <div class="card-wrapper card-teaser-wrapper card-teaser-wrapper-equal">
+                  {content.relatedItems.map((item, i) => (
+                    <GenericCard
+                      index={item['@id']}
+                      item={item}
+                      showimage={false}
+                    />
+                  ))}
+                </div>
+              </article>
+            ) : null}
```

Comment:
È normale che ci sia un'aggiunta alla vista Persona nella PR per il servizio? Se è sì perché mancava un pezzo, no problem, chiedo solo per conferma per evitare confusione.

Reply:
Si, normale. Non avevo messo i correlati. non sono nella grafica, forse nemmeno serve dai campi agid, ma i correlati in plone ci sono